websites/D/Discord Rescource/dist/metadata.json

```diff
@@ -0,0 +1,21 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"name": "2Lost4Discord",
+		"id": "421991668556759042"
+	},
+	"service": "Discord Rescource",
```

Comment:
```suggestion
	"service": "Discord Resources",
```

Reply:
This shows as outdated but it wasn't addressed

Reply:
Done